fe/hive-udf/src/main/java/com/starrocks/hive/udf/UDFBitmapCount.java

```diff
@@ -46,7 +46,7 @@ public ObjectInspector initialize(ObjectInspector[] args) throws UDFArgumentExce
 
     @Override
     public Object evaluate(DeferredObject[] args) throws HiveException {
-        if (args[0] == null) {
+        if (args[0].get() == null) {
```

Comment:
`args[0]` may be null?

Reply:
Will not be null, https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/udf/generic/GenericUDFCastFormat.java